main/telemetry.js

```diff
@@ -15,9 +20,23 @@ const writeClient = client.getWriteApi(
 )
 
 setInterval(() => {
-  writeClient.flush()
+  writeClient.flush().catch(console.error)
```

Comment:
Should we report this error to Sentry as well? See

https://github.com/filecoin-station/filecoin-station/blob/11fdc91470d864faeff5323387942d6d54a3feae/main/index.js#L47

For the longer term, it would be great to have a helper function we can use to handle errors we don't want to expose to the user.

Let's keep your current code so we can land this PR sooner and then refactor it in a follow-up PR.

Reply:
I think the change was small enough to just add on to this PR 👍 

About the helper function, for now since it's just one line of Sentry I don't feel the need to refactor this.